src/core/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,90 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { useTheme } from '../utils/hooks/useTheme';
+import { CommonProps } from '../utils/props';
+import { SvgChevronRight } from '@itwin/itwinui-icons-react';
+import '@itwin/itwinui-css/css/breadcrumbs.css';
+
+export type BreadcrumbsProps = {
+  /**
+   * Index of the currently active breadcrumb.
+   * Defaults to last index in the items array.
+   */
+  currentIndex?: number;
+  /**
+   * Breadcrumb items.
+   */
+  items: JSX.Element[];
+  /**
+   * Specifiy a custom separator element to show between breadcrumb items.
+   */
```

Comment:
Can mention that default arrow is provided

Reply:
Added